src/components/trade/LimitOrder/LimitOrder.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { LimitOrder, Props } from './LimitOrder'
+import BigNumber from 'bignumber.js'
+
+export default {
+  title: 'Trade/LimitOrder',
+} as Meta
+
+const GNO = {
+  id: 1,
+  address: '0x1',
+  symbol: 'GNO',
+  decimals: 18,
+}
+
+const DAI = {
+  id: 2,
+  address: '0x2',
+  symbol: 'DAI',
+  decimals: 18,
+}
+
+const defaultProps = {
+  sellToken: DAI,
+  receiveToken: GNO,
+  limitPrice: new BigNumber('55.247234'),
+  amount: '100',
+}
+
+const Template: Story<Partial<Props>> = (props) => {
+  const [isPriceInverted, setIsPriceInverted] = useState<boolean>(false)
+
+  return (
+    <LimitOrder
+      isPriceInverted={isPriceInverted}
+      onSwapPrices={(): void => {
+        console.log('[LimitOrder.story] Swap Prices')
+        setIsPriceInverted(!isPriceInverted)
+      }}
+      onSelectedPrice={(price): void => console.log('[LimitOrder.story] On selected price', price)}
+      onSubmitLimitOrder={(data): void => console.log('[LimitOrder.story] Submit Limit Order', data)}
+      {...defaultProps}
+      {...props}
+    />
+  )
+}
+
+export const Basic = Template.bind({})
+Basic.args = {}
+
+export const NoAmount = Template.bind({})
+NoAmount.args = {
+  amount: undefined,
```

Comment:
why `undefined`, just not passing not an option?
or does this remove `amount control` or something?

Reply:
I defined some defaults for a "normal" case. Then I just pass undefined to override that